sopel/cli/run.py

```diff
@@ -21,15 +21,20 @@
 from sopel import bot, config, logger, tools, __version__
 from . import utils
 
+# We check Python's version ourselves in case someone installed Sopel on an
+# old version of pip (<9.0.0), which doesn't know about `python_requires`.
 if sys.version_info < (2, 7):
     tools.stderr('Error: Requires Python 2.7 or later. Try python2.7 sopel')
```

Comment:
Not a problem introduced here, but backticks or words to indicate that `python2.7 sopel` is a command might be good.
Though, does that actually work, if sopel is installed to eg. python2.5's site_packages by pip? Or should it be changed to `python2.7 sopel.py`?

Reply:
I'm not going to worry about it too much. Rather than try to figure out the correct command on behalf of the user, I'd just drop the "Try" bit and let them work it out.